src/styles/items.styl

```diff
@@ -0,0 +1,68 @@
+.item
```

Comment:
Definitely not a blocking issue, but these files are a bit easier to read with an extra line between selectors. For example:

```stylus
.item
    background: lightblue
    display: block
    height: auto

    .picArea
        display: inline-block
        vertical-align: top
        padding: 7px

        img
            height: 75px
            width: inherit
```